SparkPost/2web2ui
# SA-1492: Remove region formatting

Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed
 - Removed region formatting for inbox placement.

### How To Test
 - Go to /inbox-placement and click on any test to go to details page.
 - Check that the region data in the breakdown table has not been formatted.
